bugbot/rules/not_landed.py

```diff
@@ -178,6 +178,13 @@ def comment_handler(bug, bugid, data):
                 ):
                     data[bugid]["backout"] = True
 
+                    # Set the status of the Phabricator revision to be "Changes Planned"
+                    phid = data.get("phid")
```

Comment:
Why is this not inside the conditional block since it is not used somewhere else? We may have everything inlined directly into `edit_revision()`.